unit_tests/tests/trigger/test_real_cranking_nissan_vq40.cpp

```diff
@@ -23,13 +23,13 @@ TEST(realCrankingVQ40, normalCranking) {
 		float vvt1 = engine->triggerCentral.getVVTPosition(/*bankIndex*/0, /*camIndex*/0);
 
 		if (vvt1 != 0 && !hasSeenFirstVvt) {
-			EXPECT_NEAR(vvt1, -33.204, 1);
+			EXPECT_NEAR(vvt1, -45.56, 1);
```

Comment:
@rusefillc I am VERY optimistic about this change given that these three values are now within 0.2 degree, instead of the previous 13 degree difference